lib/new_relic/agent/instrumentation/rdkafka.rb

```diff
@@ -0,0 +1,27 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative 'rdkafka/instrumentation'
+require_relative 'rdkafka/chain'
+require_relative 'rdkafka/prepend'
+
+DependencyDetection.defer do
+  named :rdkafka
+
+  depends_on do
+    defined?(Rdkafka) # TODO: check version bc some older oens break
```

Comment:
oh oops i thought i'd removed that todo, i ended up just making things compatible with older versions since the change was in a pretty recent version

Reply:
oh haha i did change it i just never pushed the changes. pushed now tho https://github.com/newrelic/newrelic-ruby-agent/pull/2824/commits/c1d8d6419eae14310e6bddde6498471849581535